src/shared/libfido2-util.c

```diff
@@ -776,10 +778,21 @@ int fido2_generate_hmac_hash(
         if (!c)
                 return log_oom();
 
-        r = sym_fido_cred_set_extensions(c, FIDO_EXT_HMAC_SECRET);
+        int extensions = FIDO_EXT_HMAC_SECRET;
+        if (FLAGS_SET(lock_with, FIDO2ENROLL_PIN) || FLAGS_SET(lock_with, FIDO2ENROLL_UV)) {
+                /* Attempt to use the "cred protect" extension, requiring user verification (UV) for this
+                 * credential. If the authenticator doesn't support the extension, it will be ignored. */
+                extensions |= FIDO_EXT_CRED_PROTECT;
+
+                r = sym_fido_cred_set_prot(c, FIDO_CRED_PROT_UV_REQUIRED);
+                if (r != FIDO_OK)
+                        log_notice("Failed to set protection level on FIDO2 credential: %s", sym_fido_strerr(r));
```

Comment:
if something fails but we ignore the failure and proceed we usually log at LOG_WARNING level, and we suffix the message with ", ignoring".

Hence, can you change this to:

```c
log_warning("Failed to set … credential, ignoring: %s", sym_fido_strerr(r));
```